### PR TITLE
Component/radio

### PR DESCRIPTION
**Pull request by driesd**

### CODE REVIEW

File: components/radio/Radio.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Radio = ({ checked, onMouseDown, theme, ...other }) => (
+  <div
+    data-react-toolbox="radio"
```
Comment (nickwaelkens) [inline components/radio/Radio.js]: ...

File: components/radio/RadioButton.js [hunk 2]
```diff
@@ -0,0 +1,112 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+import { themr } from 'react-css-themr';
+import { RADIO } from '../identifiers';
+import Radio from './Radio';
+
+const factory = (Radio) => {
```
Comment (nickwaelkens) [inline components/radio/RadioButton.js]: Are we still using `factory`?

File: components/radio/RadioButton.js [hunk 3]
```diff
@@ -0,0 +1,112 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
```
Comment (nickwaelkens) [inline components/radio/RadioButton.js]: Let's use `cx` like is the case everywhere else

File: components/index.js [hunk 4]
```diff
@@ -20,5 +21,6 @@ export {
   Overlay,
   PopoverHorizontal,
   PopoverVertical,
+  Radio,
```
Comment (nickwaelkens) [inline components/index.js]: Why not import `RadioGroup`?

**Comment by driesd** [comment]
The labels are already clickable

**Comment by nickwaelkens** [comment]
I'm kind of confused what this component looks like. There is no `label`, just the radio button?

**Comment by lowiebenoot** [comment]
@nickwaelkens RadioButton contains the label as well. Radio is just the circle.

**Pull request merged**